include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -106,14 +89,6 @@ class fpga_policy : public device_policy<KernelName>
     fpga_policy(const fpga_policy<other_factor, OtherName>& other) : base(other.queue()){};
     explicit fpga_policy(sycl::queue q) : base(q) {}
     explicit fpga_policy(sycl::device d) : base(d) {}
-
-    // For internal use only
-
-    const base&
-    __device_policy() const
-    {
-        return static_cast<const base&>(*this);
```

Comment:
I am so glad that we removed this method. It's absolute non-sense. Why do we need `using base` then?

Reply:
Fixed (removed): `base` previously has been used in constructors and etc.

Reply:
Fixed: `base` restored.